### PR TITLE
Add Chord snode

### PR DESCRIPTION
**Pull request by jaredonline**
MusicXML defines a chord by putting an empty `<chord />` inside a `<note>` object. This registers the chord type and adds it as an `snode` of `Note`.

### CODE REVIEW

**Comment by jaredonline** [comment]
From my last commit:

I was trying to get repeats to work and noticed that they use XML
attributes, so I registered them with `sattr`, but realized that looks
for a child-node and pulls the text out of for the value.

So I changed it to `prop` but realized that's a binary value, which
confused me.

This changes `prop` to actually read the attribute value from a node.

It also changes `dot` to be a `pnode` because you can have more than one
dot, and they're defined as sub-nodes on the `note` type.

**Comment by kddnewton** [comment]
Alright, now that someone else is working on this, I've added travis to run the tests and rubocop. Could you update accordingly?

File: lib/musicxml/node.rb [hunk 1]
```diff
@@ -70,5 +70,19 @@ def self.register(name, &block)
       snodes :score_instrument
     end
 
+    register :chord do
```
Comment (kddnewton) [inline lib/musicxml/node.rb]: The empty blocks here shouldn't be necessary

**Comment by jaredonline** [comment]
Awesome =D Fixed the empty blocks. I have another set of changes queued up so once this is merged I'll open another pull request.

**Pull request merged**